client/src/org/moparforia/client/Launcher.java

```diff
@@ -67,16 +67,30 @@ public static void main(String[] args) throws Exception {
         for(int i = 1; i < m.groupCount(); i++)
             System.out.println(m.group(i));
         */
-
-        String server;
-        if (args.length > 0) {
-            server = args[0];
-        } else {
-            //server = "localhost";
-            server = "127.0.0.1";
-        }
-
-        instance = new Launcher(server);
+		
+			//easily editable optional client launch options, at the moment supports custom server ip and language pack
+			//example: client.jar -server 125.456.789.123 -lang en_US
+			
+			String server = "127.0.0.1"; //dafault server ip
+			String lang = "en_US"; //dafault language package (comes with en_US, fi_FI and sv_SE)
+			
+			if(args.length==1){ //support for "legacy" .bats where only ip provided after jar
+				server=args[0];
+			}
+			
+			for(int i = 0;i<args.length ; i++)
+			{
+				if(args[i].equals("-server")){
+					server = args[i+1]; //grabs the next string after -server
+				}
+				if(args[i].equals("-lang")){
+					lang = args[i+1]; //grabs the next string after -lang
+				}
+			}
+			System.out.println(server);
```

Comment:
Please remove those 2 lines or make them more verbose than just printing a value.

Reply:
Aah this println was left in on accident when I was debugging something.